src/js/utils/checkBrowserCompatibilty.js

```diff
@@ -12,8 +12,59 @@ export function isChromium() {
     });
 }
 
+export function isAndroid() {
+    if (navigator.userAgentData) {
+        return navigator.userAgentData.mobile && navigator.userAgentData.platform === "Android";
+    }
+    return /Android/.test(navigator.userAgent);
+}
```

Comment:
For all this methods, when in capacitor, it has it's own methods to detect if Android, iOS or similar. Maybe it's better to use them here conditionally. And it will not be the same when in native mode or in browser mode.